code/modules/client/preference_setup/loadout/loadout_augments.dm

```diff
@@ -8,6 +8,12 @@
 	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD, SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI, SPECIES_SKRELL, SPECIES_SKRELL_AXIORI, SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_BULWARK, SPECIES_VAURCA_BREEDER, SPECIES_UNATHI)
 	flags = GEAR_NO_SELECTION
 
+/datum/gear/augment/power
+	display_name = "internal unified power supply"
+	description = "The central power source for all augments within a system."
+	path = /obj/item/organ/internal/augment/power
+	cost = 0
```

Comment:
this should probably be automatically added (either on spawn or on the loadout) if you have any sort of augment, otherwise I can see people forgetting to grab it